operations/phrases/dogs.js

```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
+
+module.exports = dogPhrases = async (args, message) => {
+  const supportedDogs = [
+    'dachsund', 'pug', 'schipperke', 'shiba', 'whippet'
```

Comment:
as one who prefers big fluffy dogs, can I pls request the addition of samoyed :D

Reply:
Yes, I'll add it! :)